WordPress/Classes/Utility/Editor/GutenbergSettings.swift

```diff
@@ -22,17 +27,46 @@ class GutenbergSettings {
 
     // MARK: Public accessors
 
-    func isGutenbergEnabled() -> Bool {
-        return database.object(forKey: gutenbergEditorEnabledKey) as? Bool ?? false
+    /// True if gutenberg editor is currently enabled
+    private(set) var isGutenbergEnabled: Bool {
+        get {
+            return database.bool(forKey: Key.enabled)
+        }
+        set(enabled) {
+            if enabled {
+                wasGutenbergEnabledOnce = true
+            }
+            database.set(enabled, forKey: Key.enabled)
+        }
+    }
+
+    /// True if gutenberg editor has been enabled at least once
+    var wasGutenbergEnabledOnce: Bool {
```

Comment:
I think this approach works, but I believe we could also check that `database.object(forKey: Key.enabled) != nil` to check if the value has ever been set

Reply:
Nice! Seems to work great 🎉